contrib/localnet/orchestrator/start-zetae2e.sh

```diff
@@ -42,85 +87,57 @@ sleep 2
 ### Create the accounts and fund them with Ether on local Ethereum network
 
 # unlock the default account account
-address=$(yq -r '.default_account.evm_address' config.yml)
-echo "funding deployer address ${address} with 10000 Ether"
-geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545 > /dev/null
+fund_zevm_from_config '.default_account.evm_address' 10000 "deployer"
```

Comment:
This is funding a EVM account not a ZEVM account.

Reply:
🙈 https://github.com/zeta-chain/node/pull/2816/commits/81012ce40b36eb6ad8f67799e1cb14ab033ae9d7